Add multiple sensor types for Trafikverket

endor-force opened the pull request:
**Description:**
Added precipitation instructions according to code change in sensor. Breaking change in configuration.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15935

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

endor-force committed:
Added precipitation

Added precipitation instructions according to code change in sensor.

tjorim commented on source/_components/sensor.trafikverket_weatherstation.markdown:
```diff
@@ -46,7 +47,7 @@ station:
   type: string
 type:
-  description: Defines which temperature you want (`air` or `road`).
+  description: Defines which data you want (`air`, `road` or `precipitation`).
```
Maybe we could clarify that the first two are about the temperature?

endor-force commented:
I will investigate possibilities to extend and use pytrafikverket lib..

endor-force committed:
Updated to match breaking changes

Breaking changes introduced new way of retreiving data from weatherstations using library.
Multiple sensors added and configuration needs to be update.

MartinHjelmare commented on source/_components/sensor.trafikverket_weatherstation.markdown:
````diff
@@ -29,31 +30,36 @@ sensor:
 ```
 
 {% configuration %}
-name:
````
Don't change the configuration format. See:
https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

endor-force replied:
Corrected in accordance with standards.

endor-force committed:
Corrected configuration formatting

MartinHjelmare commented on source/_components/sensor.trafikverket_weatherstation.markdown:
```diff
@@ -29,31 +30,48 @@ sensor:
+  required: true  
+  type: string  
+monitored_conditions: 
+  description: Conditions to display in the frontend.  
```
This doesn't have anything to do with the frontend per se, but decides what entities will be created for the platform, ie what states to track.

endor-force replied:
Good input, i will clarify

endor-force committed:
Clarification of monitored_conditions

MartinHjelmare reviewed (dismissed):
Looks good! If you can add `ha_iot_class` too to the front matter that would be good. It should be `cloud polling`.

endor-force committed:
Added ha_iot_class

ha_iot_class: "Cloud Polling" Added in header.

endor-force commented:
Done, don't really know why it says i dismissed your review ;)

MartinHjelmare commented:
We've set up github to automatically do that when new changes are pushed to require a new review.
